### ISSUE TITLE
errror building vignette

### ISSUE THREAD

**Issue by thibautjombart** [reconhub/epicontacts#76]
On current 3.4.0, all packages updated:
```
> check()
Updating epicontacts documentation
Loading epicontacts
Setting env vars --------------------------------------------------------------
CFLAGS  : -Wall -pedantic
CXXFLAGS: -Wall -pedantic
Building epicontacts ----------------------------------------------------------
'/usr/local/lib/R/bin/R' --no-site-file --no-environ --no-save --no-restore  \
  --quiet CMD build '/home/thibaut/dev/recon/epicontacts' --no-resave-data  \
  --no-manual 

* checking for file ‘/home/thibaut/dev/recon/epicontacts/DESCRIPTION’ ... OK
* preparing ‘epicontacts’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
Warning in structure(x, class = unique(c("AsIs", oldClass(x)))) :
  Calling 'structure(NULL, *)' is deprecated, as NULL cannot have attributes.
  Consider 'structure(list(), *)' instead.
Quitting from lines 219-233 (epicontacts.Rmd) 
Error: processing vignette 'epicontacts.Rmd' failed with diagnostics:
undefined columns selected
Execution halted
Error: Command failed (1)
```

**Comment by thibautjombart** [comment]
@vpnagraj Found it. Changes in the new release of *outbreaks*. Columns in `ebola_sim` used to be using 'wrong' case. Now uses snake_case. I'll make the adjustments and PR.

**Comment by thibautjombart** [comment]
Fixed via bd685c2167a4b8621b790415bec4ca9e5eccf0fc
